data/json/monstergroups.json

```diff
@@ -9,7 +9,14 @@
     "type":"monstergroup",
     "name" : "GROUP_FOREST",
     "default" : "mon_null",
-    "monsters" : [
+    "monsters" : [ { "monster" : "mon_wild_beehive", "freq" : 1, "cost_multiplier" : 10, "pack_size" : [1,1], "conditions" : ["DAY","DUSK","NIGHT","DAWN","AUTUMN", "WINTER"] },
+      { "monster" : "mon_wild_beehive_empty", "freq" : 1, "cost_multiplier" : 20, "pack_size" : [1,1], "conditions" : ["DAY","DUSK","NIGHT","DAWN","SPRING"] },
```

Comment:
There is no monster type "mon_wild_beehive_empty" in the game (without this mod). But this file will be read and used independent of your mod. Playing in a world without your mod still loads this file as it is and therefor loads monster groups that contain "mon_wild_beehive_empty", which will not be defined.